wodles/aws/aws_s3.py

```diff
@@ -1084,14 +1091,24 @@ def check_bucket(self):
             else:
                 print("ERROR: No files were found in '{0}'. No logs will be processed.".format(self.bucket_path))
                 exit(14)
-        except botocore.exceptions.ClientError as err:
-            if err.response['Error']['Code'] == 'ThrottlingException':
-                debug(f'Error: The "check_bucket" request was denied due to request throttling. ', 2)
-                sys.exit(16)
-            else:
-                print("ERROR: Invalid credentials to access S3 Bucket")
-                exit(3)
 
+        except botocore.exceptions.ClientError as error:
+            error_message = "Unknown"
+            exit_number = 1
+            error_code = error.response.get("Error", {}).get("Code")
+
+            if error_code == THROTTLING_EXCEPTION_ERROR:
+                error_message = "The 'check_bucket' request was denied due to request throttling."
```

Comment:
To be consistent with the rest of your changes this message should be placed in a constant, too. You should create something like `THROTTLING_EXCEPTION_ERROR_MESSAGE` and set this to:
```suggestion
                error_message = f"{THROTTLING_EXCEPTION_ERROR_MESSAGE}: {error}"
```
Apply this change accordingly every time a throttling exception is being handled in the rest of the module.